Model/Import/AbstractRatesConsumer.php

```diff
@@ -134,28 +138,18 @@ public function process(\Magento\AsynchronousOperations\Api\Data\OperationInterf
     /**
      * Reset member variables as class is not reinstantiated between operations
      */
-    private function reset()
+    protected function reset()
     {
         $this->newRates = [];
         $this->newShippingRates = [];
     }
 
-    /**
-     * @throws LocalizedException
-     */
-    private function processOperation()
-    {
-        $this->validate();
-        $this->setData();
-        $this->createRatesAndRules();
-    }
-
     /**
      * Validate that backup rates should be imported
      *
      * @throws LocalizedException
      */
-    private function validate()
+    protected function validate()
```

Comment:
We may have to move this validate method to the create rates consumer. In the current version, when a customer disables the backup rates, or removes their connection to TaxJar, the delete rates consumer would never actually delete the rates due to this validation step and the rates would not be cleaned up.